### ISSUE TITLE
Compact Multi-Class Boosted Trees configuration 

### ISSUE THREAD

**Issue by samanemami** [tensorflow/tensorflow#48154]
I am trying implement the TFBT algorithm from the [Compact multi-class boosted trees](https://ieeexplore.ieee.org/abstract/document/8257910?casa_token=OcIH95osyb4AAAAA:OwLjp3Fs9BbWlYGx_mtC96HRjaDU2311UxfgEjpJCocDCKSbXUaKdOaQnEBwT9u8BfrZefYqNA) paper. As I understand this [paper](https://ieeexplore.ieee.org/abstract/document/8257910?casa_token=OcIH95osyb4AAAAA:OwLjp3Fs9BbWlYGx_mtC96HRjaDU2311UxfgEjpJCocDCKSbXUaKdOaQnEBwT9u8BfrZefYqNA) used the [BoostedTreeClassifier](https://github.com/tensorflow/estimator/blob/781c0d30c6bf100aa174591dd97cb70fc39d294d/tensorflow_estimator/python/estimator/canned/boosted_trees.py#L1933) with [this](https://github.com/tensorflow/estimator/blob/781c0d30c6bf100aa174591dd97cb70fc39d294d/tensorflow_estimator/python/estimator/canned/boosted_trees_test.py#L403) config.

- Would you please indicate the same **setting** and **hyperparameter** which you used in your paper?
- Also, would you verify that the mentioned paper used the above code?
- How can I have Fit method in [BoostedTreeClassifier](https://github.com/tensorflow/estimator/blob/781c0d30c6bf100aa174591dd97cb70fc39d294d/tensorflow_estimator/python/estimator/canned/boosted_trees.py#L1933)?

**Comment by rmothukuru** [comment]
@samanemami,
The usage of **`Boosted Trees Using Estimators`** is demonstrated in the [Tensorflow Tutorial](https://www.tensorflow.org/tutorials/estimator/boosted_trees) . It is further elaborated in [this Tutorial](https://www.tensorflow.org/tutorials/estimator/boosted_trees_model_understanding).

If this is not what you are looking for, can you please elaborate your point. Thanks!

**Comment by samanemami** [comment]
I have read this [Tutorial](https://www.tensorflow.org/tutorials/estimator/boosted_trees). The thing is I need to be sure that this is the same model that you used in this [paper](https://ieeexplore.ieee.org/abstract/document/8257910?casa_token=OcIH95osyb4AAAAA:OwLjp3Fs9BbWlYGx_mtC96HRjaDU2311UxfgEjpJCocDCKSbXUaKdOaQnEBwT9u8BfrZefYqNA).
Also, I need to have the configuration of the hyper-parameter which you used in this [paper](https://ieeexplore.ieee.org/abstract/document/8257910?casa_token=OcIH95osyb4AAAAA:OwLjp3Fs9BbWlYGx_mtC96HRjaDU2311UxfgEjpJCocDCKSbXUaKdOaQnEBwT9u8BfrZefYqNA).

**Comment by rmothukuru** [comment]
@samanemami,
The Tutorial, [Boosted trees using Estimators](https://www.tensorflow.org/tutorials/estimator/boosted_trees) states, 

> This tutorial is an end-to-end walkthrough of training a Gradient Boosting model using decision trees with the tf.estimator API. 

So, we can say that the Model used in the [paper](https://ieeexplore.ieee.org/abstract/document/8257910?casa_token=OcIH95osyb4AAAAA:OwLjp3Fs9BbWlYGx_mtC96HRjaDU2311UxfgEjpJCocDCKSbXUaKdOaQnEBwT9u8BfrZefYqNA) is same as in [the Tutorial](https://www.tensorflow.org/tutorials/estimator/boosted_trees) because both are related to **`Gradient Boosting Estimators`**.

Please refer [this code](https://github.com/tensorflow/estimator/blob/781c0d30c6bf100aa174591dd97cb70fc39d294d/tensorflow_estimator/python/estimator/canned/boosted_trees.py#L1096-L1405) for the Configuration of the Hyper-Parameters. Thanks!
